php/php.editor/src/org/netbeans/modules/php/editor/indent/ui/Bundle.properties

```diff
@@ -430,3 +430,7 @@ FmtWrapping.groupUseListLabel.text=G&roup Use List:
 FmtWrapping.groupUseBracesCheckBox.text_1=Group &Use Braces
 FmtBraces.groupUseLabel.text=G&roup Use:
 FmtAlignment.noteLabel.text=<html><i>Note: These options depend on Wrapping settings</i></html>
+FmtBlankLines.maxPreservedField.text=
+FmtBlankLines.maxPreservedLabel.text=Max Preserved Lines:
```

Comment:
Please, add "blank" everywhere here as it is not apparent what exactly it does. The constant as well as the Symfony rule contains it. Thanks.

CC @junichi11

Reply:
Changed name to `maxPreservedBlankFiled` and `maxPreservedBlankLabel`.

Reply:
Well, I was talking about the text of the label, not the label name. But thanks for changing it too and sorry for not being clear. Please, improve the label so it is obvious what lines are going to be preserved. Thanks!

Reply:
@tmysik Maybe, I suppose @KacerCZ did not add it because Category is Blank Lines.

![nb2573-formatting-option](https://user-images.githubusercontent.com/738383/101270099-17a37d80-37b9-11eb-9367-ad895063c183.png)

Reply:
Changing text to "Max Preserved Blank Lines:" widens panel.
I hope meaning of this setting will be clear from category as @junichi11 says.

Reply:
I see now... OK then. Thanks!